packages/lesswrong/components/common/SectionTitle.tsx

```diff
@@ -58,7 +58,8 @@ export const getAnchorId = (anchor: string|undefined, title: React.ReactNode) =>
 
 export type SectionTitleProps = {
   children?: React.ReactNode,
-  className?: string,
+  titleClassName?: string,
+  rootClassName?: string,
```

Comment:
renaming the `className` prop requires that you change all uses of it by other components (for `SectionTitle`, it seems like there are a lot).  tsc should have been throwing type errors at you about this

Reply:
k, made a commit changing all 18 instances of this in the codebase